common/_overrides.scss

```diff
@@ -19,11 +19,12 @@
 }
 
 // Set the default color of links
-a:not(.btn, .help-link),
-a:not(.btn):hover,
-a:not(.btn):focus,
-a:visited:not(.btn):hover,
-a:visited:not(.btn):focus {
+// HACK: The first rule splits classes into separate :not selectors to increase the specificity.
+a[href]:not(.btn):not(.help-link),
+a[href]:not(.btn):hover,
+a[href]:not(.btn):focus,
+a[href]:visited:not(.btn):hover,
+a[href]:visited:not(.btn):focus {
```

Comment:
E.g., the Account MFE is a selector with higher specificity at the bottom of the page. While we could add `!important` here, this would make further overrides harder.

Reply:
@xitij2000, good idea. I created [BB-7109](https://tasks.opencraft.com/browse/BB-7109).

Reply:
@xitij2000, good idea. I created [BB-7109](https://tasks.opencraft.com/browse/BB-7109).